include/vapor/ConstantGrid.h

```diff
@@ -0,0 +1,85 @@
+#ifndef CONSTANTGRID_H
+#define CONSTANTGRID_H
+
+/* 
+ * This class represents a constant field. That means,
+ * querying values from any location of this field will return that constant value. 
+ *
+ * It also implements pure virtual functions of the Grid class in the simplest possible fashion,
+ * but those functions do not perform any calculation, and should not be used.
+ *
+ * Finally, a DataMgr would not return a ConstantGrid at any occasion; rather, this 
+ * class is supposed to be created and used locally by its user.
+ */
+
+#include "vapor/Grid.h"
+
+namespace VAPoR
+{
+
+class VDF_API ConstantGrid : public Grid
+{
+public:
+
+    // The constant value is specified via the constructor
```

Comment:
The constant grid is also a dimensionless grid. I wonder if that is what we want, or if it would be better to specify dimensions, and coordinates? Maybe instead of being derived from Grid it should be derived from StructuredGrid (or RegularGrid). Something to consider for the future if we use ConstantGrid elsewhere.

Reply:
I guess we can have create derivatives of `ConstantGrid` that represent `ConstantStructuredGrid` and `ConstantRegularGrid`, and each one of them will have more specific information held by them?

Reply:
At some point we want to make ConstantGrid more like a real Grid class: have all of the public methods do reasonable things so that a ConstantGrid instance can easily be used anywhere we are using a Grid class. For now I'd say let's just move on since the current implementation is meeting our needs for flow visualization.